network/containerizer/bridgepolicy_test.go

```diff
@@ -1,1174 +1,97 @@
-// Copyright 2017 Canonical Ltd.
+// Copyright 2019 Canonical Ltd.
 // Licensed under the AGPLv3, see LICENCE file for details.
 
-package containerizer_test
+package containerizer
```

Comment:
This one is best viewed in isolation. It is essentially a new file - the old contents were relocated.